frontend/src/app/supportAdmin/DeviceType/ManageDevicesForm.tsx

```diff
@@ -113,6 +114,32 @@ const ManageDevicesForm: React.FC<Props> = ({
                     required
                   />
                 </div>
+              </div>
+              <div className="grid-row grid-gap">
+                <div className="tablet:grid-col">
+                  <TextInput
+                    label="Device name"
+                    name="deviceName"
+                    value={selectedDevice?.name}
+                    onChange={onChange}
```

Comment:
I believe this will not update the device name properly. Check out the `onChange` handler that we're passing in:

https://github.com/CDCgov/prime-simplereport/blob/760a379c309badedd6c5ed8ec8c74a6e064d11be/frontend/src/app/supportAdmin/DeviceType/ManageDevicesForm.tsx#L37-L41

The handler uses the `name` property of the target HTML element to know which device attribute to update. However, this input is named `deviceName` and there is no corresponding attribute for a device.